angularjs-portal-frame/src/main/webapp/js/app-config.js

```diff
@@ -17,6 +17,10 @@ define(['angular'], function(angular) {
             'crest' : 'img/uwcrest_web_sm.png',
             'crestalt' : 'UW Crest'
         })
+        .constant('SEARCH',{
+            'isWeb' : false,
```

Comment:
I wonder if web-ness is going to end up being a concept applicable more broadly than in search.  I'm fine with it starting here where angularjs-portal is prepared to take advantage of the concept today, and refactor it out if it ever turns out to be a wider thing.